superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -658,6 +672,14 @@ class Header extends React.PureComponent {
             />
           )}
 
+          {isFeatureEnabled(FeatureFlag.EMBEDDED_SUPERSET) && (
```

Comment:
should `userCanCurate` be used here instead just the feature flag check?

Reply:
oh yeah, I moved the ff but forgot to remove it from the original spot